portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const SPOTIFY_LINK = "https://open.spotify.com/playlist/6rmiBWfdA4jGdTWaYMAQri";
+
+
+/**
+ * Sends user a spotify link
+ */
+function sendSpotifyLink(){
+    const linkContainer = document.getElementById("spotify-link-container");
```

Comment:
So this will give them the link no matter what? From the description, sounded like you only wanted to show this if they got it right

Reply:
Fixed this.